distributed/tests/test_as_completed.py

```diff
@@ -65,6 +65,22 @@ def test_as_completed_add(loop):
             assert total == expected
 
 
+def test_as_completed_update(loop):
+    with cluster() as (s, [a, b]):
+        with Client(s['address'], loop=loop) as c:
+            total = 0
+            todo = list(range(10))
+            expected = sum(map(inc, todo))
+            ac = AsCompleted([])
```

Comment:
FWIW we've been using the `as_completed` syntax everywhere.  Actually, I wouldn't mind renaming the class to `as_completed`.